skippylab/instruments/function_generators.py

```diff
@@ -107,9 +107,18 @@ def __init__(self, ip="10.25.124.252", gpib_address=10, loglevel=20):
             gpib_address (int): The GPIB address of the power supply
                                 connected to the Prologix connector
         """
-        gpib = PrologixGPIBEthernet(ip, timeout=10)
-        gpib.connect()
-        gpib.select(gpib_address)
+
+        try:
+            gpib = PrologixGPIBEthernet(ip, timeout=15)
+            gpib.connect()
+        except ImportError:
+            raise ValueError('Connection to plx_gpib_ethernet failed, check connection!')
+
+        try:
+            gpib.select(gpib_address)
+        except ImportError:
+            raise ValueError('Connection to plx_gpib_ethernet failed, check IP address!')
```

Comment:
Are you sure an `ImportError` is the right thing to catch here? I would guess it is rather unlikely that this type of Error will happen here.